doc/plugin_agent_workloadattestor_k8s.md

```diff
@@ -31,6 +31,9 @@ server name validation against the kubelet certificate.
 > mitigate this issue. A large cache ttl value is not recommended however, as
 > that can impact permission revocation.
 
+**Note** To run on windows containers, kubernetes v1.24+ and containerd v1.6+ are required,
+since [hostprocess](https://kubernetes.io/docs/tasks/configure-pod-container/create-hostprocess-pod/) is required on agent container
```

Comment:
```suggestion
since a [HostProcess] container (https://kubernetes.io/docs/tasks/configure-pod-container/create-hostprocess-pod/) is required on the agent container.
```